TaskLayer/SearchTask/MzIdentMLWriter.cs

```diff
@@ -682,7 +682,7 @@ public static void WriteMzIdentMl(IEnumerable<PeptideSpectralMatch> psms, List<E
                                 accession = "MS:1001093",
                                 name = "sequence coverage",
                                 cvRef = "PSI-MS",
-                                value = proteinGroup.SequenceCoveragePercent.First().ToString()
+                                value = proteinGroup.SequenceCoverageFraction.First().ToString()
```

Comment:
Does MzIdent expect an integer percent here? This might break something down the line if so

Reply:
i don't know. i'll have to check

Reply:
👍

Reply:
the ask for a decimal value. I checked and other terms require integer or positive integer. I also saw examples of decimal value.